sherpa-onnx/csrc/online-recognizer.h

```diff
@@ -162,6 +162,8 @@ class OnlineRecognizer {
     DecodeStreams(ss, 1);
   }
 
+  void WarmpUpRecognizer(int32_t warmup, int32_t mbs) const;
```

Comment:
Please leave a comment describing what `mbs` is.

(I know it is short for max batch size, but I only know it after reading the code)

Reply:
yes, will add this